### PR TITLE
added missing address info on iOS to card nonce

### PR DESCRIPTION
**Pull request by TGier**
We need to manually add all the fields like we do on Android.
Tested via making this change locally in my Xcode project and submitting (and succeeding) on a address required and address not-required deposit.

### CODE REVIEW

**Review by PaigeKH — APPROVED** [review]
🐉

File: ios/RCTBraintree/RCTBraintree.m [hunk 1]
```diff
@@ -121,8 +121,6 @@ - (void)handlePayPalResult:(BTPayPalAccountNonce * _Nullable)tokenizedPayPalAcco
         }
     } else if ( error != nil ) {
         args = @[error.description, [NSNull null]];
-    } else { // per braintree docs, if both error and token are nil, user cancelled
-        args = @[@"USER_CANCELLATION", [NSNull null]];
```
Comment (pdrlima) [inline ios/RCTBraintree/RCTBraintree.m]: is it ok to remove this? did you test what happens when you just cancel without depositing?

Comment (TGier) [reply ios/RCTBraintree/RCTBraintree.m]: Nope, i didn't catch that got changed. I think some stuff got messed up between pulling from master and such. Will fix it

**Review by pdrlima — APPROVED** [review]
👍

**Pull request merged**